Technical-Workshop/Presentation/Home Screen/View/HomeScreenViewController.swift

```diff
@@ -10,46 +10,80 @@ import UIKit
 class HomeScreenViewController: UIViewController {
     
     @IBOutlet weak var categoriesCollectionView: UICollectionView!
-    @IBOutlet weak var homeTableView: UITableView!
+    @IBOutlet weak var mealsCollectionView: UICollectionView!
     
     private lazy var viewModel: HomeScreenViewModelType = {
         HomeScreenViewModel()
     }()
         
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        mealsCollectionView.register(UINib(nibName: "MealCollectionViewCell", bundle: .main), forCellWithReuseIdentifier: "MealCollectionViewCell")
```

Comment:
could we use some thing else instead of the raw strings as it's more prone to errors